projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -352,12 +352,14 @@ public static function parse( $attributes, $content ) {
 			$url                     = apply_filters( 'grunion_contact_form_form_action', "{$url}#contact-form-{$id}", $GLOBALS['post'], $id );
 			$has_submit_button_block = str_contains( $content, 'wp-block-jetpack-button' );
 			$form_classes            = 'contact-form commentsblock';
+			$form_accessible_name    = $attributes['formTitle'];
```

Comment:
Recycling `formTitle` here. Is there any side effect to renaming a block attribute?

Reply:
Not sure here. We are not using it anywhere else, so looks good. But unsure if there is some type of contract for blocks and their attributes.